azurerm/helpers/validate/url.go

```diff
@@ -0,0 +1,48 @@
+package validate
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"strings"
+)
+
+func UrlIsHttpOrHttps(i interface{}, k string) (_ []string, errors []error) {
```

Comment:
There may have been a precedent here you are following, but traditionally initialisms in Go are capitalized: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

`URLIsHTTPOrHTTPS` I think? I may be misreading this thing from Go, may want to run the linter on it.